### PR TITLE
Added a check to ignore broadcasts in cluster when not holding the VIP.

### LINKED ISSUE #2408: DHCP broadcast packets should not be handled by all servers in a cluster
Currently, all pfdhcplistener processes handle packets.
We should be treating broadcast requests differently from unicast ones.

### PR DESCRIPTION
**Pull request by louismunro**
# Description
Adds a check to pfdhcplistener to ignore broadcast dhcprequests in clustered mode if this server is hot currently holding the VIP.

# Impacts
DHCPv4 requests to the broadcast ip (255.255.255.255) should be ignored in a cluster when the node does not hold the VIP.

# Issue
fixes #2408 

# Delete branch after merge
YES 

# NEWS file entries

## Bug Fixes
If an issue exists on Github, please refer to it (name) along with it's number...
* Fixed issue where dhcp broadcast requests are handled multiple times in a cluster (#2408)

### CODE REVIEW

File: sbin/pfdhcplistener [hunk 1]
```diff
@@ -107,6 +109,8 @@ sub reload_config {
     }
     elsif ( pf::cluster::is_vip_running($interface) ) { 
         $process = $TRUE;
+        $is_clustered = $TRUE;
+        $holds_vip = $TRUE;
```
Comment (julsemaan) [inline sbin/pfdhcplistener:113]: You're never setting this variable back to false so it seems that if you own the VIP once, this flag will stay true even if you lose the VIP

**Comment by julsemaan** [comment]
Also, IMO this belongs in maintenance

File: sbin/pfdhcplistener (hunk 1 above)
Comment (louismunro) [reply sbin/pfdhcplistener:113]: You can get the VIP back when you pry it from my cold, dead hands...

All right, I'll give it back...
Fixing this.

**Comment by louismunro** [comment]
Agreed, this should be cherry picked.

**Pull request merged**